web/styles/compose.css

```diff
@@ -173,21 +173,15 @@
     height: 100%;
 }
 
-#below-compose-content {
-    display: flex;
-    flex-direction: column;
-    width: 100%;
+.compose_bottom_top_container {
     margin-top: 6px;
-    margin-bottom: -2px;
```

Comment:
Outside the PR at present, but if that 6px margin is showing up elsewhere in the compose box, we ought to specify its purpose as a variable, and then reference the variable in spots like this.